created hello world with java

davenbezz opened the pull request:
the main file is hello.java others are created by the IDLE

davenbezz committed:
created hello world with java

the main file is hello.java others are created by the IDLE

michaelbelete reviewed (changes requested):
make the first class name helloWorld instead of hello its more clear

davenbezz committed:
change have made

I've changed the hello.java file to helloworld.java and also the public class of the java file is changed to helloworld.

davenbezz commented:
https://docs.google.com/forms/d/e/1FAIpQLSfnt9e7zoAGkPuDrsA1tQgQhJYQRXUvdKryCmptczNKoec25Q/viewform?fbclid=IwAR3CgMVLW-cDFf57Br5awAcM9dSG1UqUsDsLIzTCp5ILwOTSQmim9pJvDrE

Sent from Mail for Windows 10

From: Michael Belete
Sent: Friday, October 25, 2019 1:56 PM
To: michaelbelete/TeamAlpha
Cc: davenbezz; Author
Subject: Re: [michaelbelete/TeamAlpha] created hello world with java (#1)

Merged #1 into master.
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.



-- 
This email has been checked for viruses by Avast antivirus software.
https://www.avast.com/antivirus
